lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -133,12 +135,15 @@ defmodule ExUnit.Assertions do
   defmacro assert({:match?, meta, [left, right]} = assertion) do
     code   = Macro.escape(assertion)
     match? = {:match?, meta, [left, Macro.var(:right, __MODULE__)]}
+    pins = collect_pins_from_pattern(left)
```

Comment:
Last tiny nitpick: seems we should align `=` to same level as previous ones.
And same in `refute`.